_posts/2016-11-10-JavaScript-Event-Flow.md

```diff
@@ -18,21 +18,37 @@ Experiment with [this](https://gist.github.com/domfarolino/fdde99c1ad3fa1668a184
 
 # Event Flow
 
-When you click an element that is nested in various other elements, before your `click` actually reaches its destination, or [`target element`](https://www.w3.org/TR/DOM-Level-3-Events/#event-target), it must trigger the click event each of its parent elements first, starting at the top with the global `window` object. The click event trickles down parent by parent in what is called the [`capture phase`](https://www.w3.org/TR/DOM-Level-3-Events/#capture-phase) until it finally arrives at the event's intended target element.
```

Comment:
@KyTrol Take a look at some of the changes I made to this article and let me know what you think.